src/main/java/com/kaziamyr/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package com.kaziamyr.onlinebookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
```

Comment:
Do you really need this class, maybe it is possibe to reuse BookDto they have almost all same fields

Reply:
I also think that you could reuse BookDto, but its up to you

Reply:
https://mateacademy.slack.com/archives/C0554BPUYJ1/p1692708245197979